linux_os/guide/system/software/integrity/software-integrity/aide/aide_periodic_cron_checking/rule.yml

```diff
@@ -80,11 +102,26 @@ ocil: |-
     <pre>05 4 * * * root {{{ aide_bin_path }}} {{{ config_command_line }}}--check</pre>
 
     NOTE: The usage of special cron times, such as @daily or @weekly, is acceptable.
+{{%- endif %}}
 
 fixtext: |-
+    Configure the file integrity tool to run automatically on the system at least weekly and to
+    notify designated personnel if baseline configurations are changed in an unauthorized manner.
+    {{%- if "rhel" in product %}}
```

Comment:
What is special about RHEL here?

Reply:
The alternative form that used the `config_command_line` was introduced by Ubuntu. And as it doesn't align with the DISA RHEL9 content, I wouldn't remove it entirely - it seems to be clever.